main/main.c

```diff
@@ -45,6 +47,22 @@ uint64_t cycles_per_sec;
 uint64_t cycles_per_ms;
 uint64_t picosec_per_cycle;
 
+FILE *log_file;
+
+static int
+gatekeeper_log_init(void)
+{
+	log_file = fopen(LOG_FILE_PATH, "w+");
+	if (log_file == NULL) {
+		RTE_LOG(ERR, GATEKEEPER,
+			"gatekeeper: Failed to open log file %s - %s\n",
+			LOG_FILE_PATH, strerror(errno));
+		return -1;
+	}
+
+	return rte_openlog_stream(log_file);
```

Comment:
This patch doesn't close issue #76 since as it's explained in the issue: "Remember that Gatekeeper is geared toward production, so having a way to name the log files and describing how to break up the log files (e.g. per hour, per minute) in the configuration of Gatekeeper is essential to complete this issue."

Reply:
Thanks @AltraMayor . I have explored some existing solutions using log file. However, all of them are directly using one predefined file to dump the log. I will try to add a configuration so that Gatekeeper will change the log file every a time interval.